File: ruby/lib/game_of_life/cell.rb
module GameOfLife

    #Represents a live cell
  class Cell
    attr_accessor :x
    attr_accessor :y
    
    def initialize( x, y )
      @x = x
      @y = y
    end
    
    def ==( other )
      @x == other.x && @y == other.y
    end
    
  end

end

File: csharp/GameOfLife/Cell.cs
using System;

namespace GameOfLife
{
    public class Cell
    {
        public int X { get; set; }
        public int Y { get; set; }

        public Cell(int xCoord, int yCoord)
        {
            this.X = xCoord;
            this.Y = yCoord;
        }

        public bool EqualTo(Cell other) 
        {
            return (this.X == other.X) && (this.Y == other.Y);
        }
    }
}
